baseimages/docker.Windows.Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Required.
```

Comment:
nit: usually the convention that I saw for windows docker files is Dockerfile.Windows , but it is fine too

Reply:
@mnltejaswini I renamed all of them to end in `Dockerfile` so that you can get syntax highlighting in things like VSCode. You don't get this unless it's a suffix.